lib/middleware/request-handler.ts

```diff
@@ -212,9 +171,9 @@ export function onSubscribe(req: Interface.IOurRequest,
         }
 
         req.body.forEach((s: {'correlationId': number;
-                              'security': string;
-                              'fields': string[];
-                              'options'?: any }): void => {
+            'security': string;
```

Comment:
For this change I'll preserve the indentation that existed (i.e. this code got reformatted due to my editor not preserving the indent when I moved this function). If we want to introduce a type alias we can do that separately.